bigtable/admin.go

```diff
@@ -275,6 +278,35 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// UpdateTable updates a table in the instance from the given configuration.
+// Only deletion protection can be updated at this moment
```

Comment:
Nit: Only deletion protection can be updated at this moment.

End a sentence with a period.